Library/Homebrew/os/linux/elf.rb

```diff
@@ -204,7 +237,22 @@ def patchelf_patcher
     @patchelf_patcher ||= begin
       Homebrew.install_bundler_gems!
       require "patchelf"
-      PatchELF::Patcher.new to_s, logging: false
+
+      patcher = PatchELF::Patcher.new to_s, logging: false
+      class << patcher
+        def rpath
+          super
+        rescue PatchELF::MissingTagError
+          nil
+        end
+
+        def runpath
+          super
+        rescue PatchELF::MissingTagError
+          nil
+        end
+      end
+      patcher
```

Comment:
Neat!
@MikeMcQuaid Do you like this style?

Reply:
I'm 👍 on this style and patch for now, although I also agree with the Slack convo about getting this upstreamed.

Reply:
@MikeMcQuaid wrote…
> Please use refinements for this (and submit PRs to change the upstream software ideally).